ansible/roles/ironic/templates/ironic-ipxe.json.j2

```diff
@@ -0,0 +1,5 @@
+{% set apache_cmd = 'apache2' if kolla_base_distro in ['ubuntu', 'debian'] else 'httpd' %}
+
+{
+    "command": "{{ apache_cmd }} -DFOREGROUND -f {{ container_config_directory }}/httpd.conf"
```

Comment:
It's normal in kolla to have an entry in here to copy the config file into place rather than use container_config_directory. That means changes to the file on the host aren't picked up until the container restarts.

Reply:
shall I overwrite the config installed by the package manager? or just copy it to `/etc/httpd/conf/httpboot.conf`?

Reply:
See what keystone does for wsgi-keystone: ansible/roles/keystone/templates/keystone.json.j2